packages/testlab/should-as-function.d.ts

```diff
@@ -118,15 +118,18 @@ interface ShouldAssertion {
   enumerables(...properties: string[]): ShouldAssertion;
   startWith(expected: string, message?: any): ShouldAssertion;
   endWith(expected: string, message?: any): ShouldAssertion;
-  throw(message?: any): ShouldAssertion;
+  throw(): ShouldAssertion;
+  throw(message: Function | string | RegExp, properties?: {}): ShouldAssertion;
```

Comment:
Doesn't having both `message` and `properties` as options accomplish the same thing as defining 3 different definitions.

Reply:
Honestly, I didn't give it too much thought. All I did was essentially copy over the definitions from `should-js` itself: https://github.com/shouldjs/should.js/blob/master/should.d.ts#L119

Reply:
> Doesn't having both message and properties as options accomplish the same thing as defining 3 different definitions.

I agree we could join `throw()` and `throw(properties: {})` into a single definition. I think the second one with `message` argument will be always needed.